src/employeeProfiles.js

```diff
@@ -78,10 +115,33 @@ const GetLatestAssignment = () => {
   return latestAssignment;
 };
 
-const CustomEmpty = () => <div>No vacations found</div>;
+const CustomEmpty = ({ message }) => <div>{message}</div>;
+
+const style = {
+  position: "absolute",
+  top: "50%",
+  left: "50%",
+  transform: "translate(-50%, -50%)",
+  width: 400,
+  bgcolor: "background.paper",
+  border: "2px solid #000",
+  boxShadow: 24,
+  p: 4,
+};
 
 export const EmployeeProfile = () => {
   const [locale] = useLocaleState();
+  const data = GetVacationDetail();
+  const listContext = useList({ data });
+  const availableDays = GetAvailableDays();
```

Comment:
I would rename this variable names:
data => vacationDetailData
listContext => vacationDetailList
availableDays => vacationAvailableDays 

I see we call `GetVacationDetail` then `GetAvailableDays` where internally calls `GetVacationDetail` again. is it doing 2 calls to the backend?